packages/test/test-end-to-end-tests/src/test/incrementalSummaryContext.spec.ts

```diff
@@ -543,9 +559,27 @@ describeCompat(
 			);
 			const ddsTree = dataObjectChannelsTree.tree[dds.id];
 			assert(ddsTree.type === SummaryType.Tree, "Blob dds tree not created");
-			assert(ddsTree.tree["0"].type === SummaryType.Handle, "Blob 0 should be a handle");
-			assert(ddsTree.tree["1"].type === SummaryType.Handle, "Blob 1 should be a handle");
-			assert(ddsTree.tree["2"].type === SummaryType.Handle, "Blob 2 should be a handle");
+			validateHandle(
+				ddsTree.tree["0"],
+				datastore.context.id,
+				dds.id,
+				"0",
+				"Blob 0 handle is incorrect",
```

Comment:
>incorrect [](http://example.com/codeflow?start=22&length=9)

Is the older message better, as it just says what's expected! Depending on how this string shows up in the log, it can be confusing to the reader, if Blob0 handle validation succeeded.

Reply:
Updated the messaging to reflect what failed.